src/serac/physics/materials/functional_material_utils.hpp

```diff
@@ -198,50 +188,57 @@ auto parameterizeFlux(T& flux)
   }
 }
 
-template <typename T, int dim, typename = void>
-struct has_stress : std::false_type {
-};
-
-template <typename T, int dim>
-struct has_stress<T, dim, std::void_t<decltype(std::declval<T&>()(std::declval<tensor<double, dim, dim>&>()))>>
-    : std::true_type {
-};
-
-// Use SFINAE to add static assertions checking if the given thermal source type is acceptable
-template <typename T, int dim, typename = void>
-struct has_body_force : std::false_type {
-};
-
-template <typename T, int dim>
-struct has_body_force<T, dim,
-                      std::void_t<decltype(std::declval<T&>()(
-                          std::declval<tensor<double, dim>&>(), std::declval<double>(),
-                          std::declval<tensor<double, dim>&>(), std::declval<tensor<double, dim, dim>&>()))>>
-    : std::true_type {
-};
-
-// Use SFINAE to add static assertions checking if the given thermal flux boundary type is acceptable
-template <typename T, int dim, typename = void>
-struct has_traction_boundary : std::false_type {
-};
+/**
+ * @brief Wrapper to treat an unparameterized source as a parameterized one
+ *
+ * @tparam UnparameterizedSourceType The unparameterized source
+ */
+template <typename UnparameterizedPressureType>
+struct TriviallyParameterizedPressure {
+  /**
+   * @brief Wrapper for an unparameterized source in a parameterized context
+   *
+   * @tparam T1 Spatial position type
```

Comment:
If `T2` and `T3` aren't arguments to this template, is it necessary to document them?

Reply:
Good point. I'm surprised our doxygen bot didn't notice this.